packages/e2e-tests/src/tests/SessionSigs.spec.ts

```diff
@@ -0,0 +1,825 @@
+  //   publicKey: "04400AD53C2F8BA11EBC69F05D1076D5BEE4EAE668CD66BABADE2E0770F756FDEEFC2C1D20F9A698EA3FEC6E9C944FF9FAFC2DC339B8E9392AFB9CC8AE75C5E5EC",
+  //   dataSigned: "7D87C5EA75F7378BB701E404C50639161AF3EFF66293E9F375B5F17EB50476F4",
+  // }
+
+  expect(res?.r).toBeDefined();
+  expect(res?.s).toBeDefined();
+  expect(res?.dataSigned).toBeDefined();
+  expect(res?.publicKey).toBeDefined();
+};
+
+const executeJsJSONResponse = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
+
+  const alice = await devEnv.createRandomPerson();
+  const litActionSessionSigs = await generator(devEnv, alice);
+
+  const res = await devEnv.litNodeClient?.executeJs({
+    sessionSigs: litActionSessionSigs,
+    code: `(async () => {
+    console.log('hello world')
+
+    LitActions.setResponse({
+      response: JSON.stringify({hello: 'world'})
+    });
+
+  })();`,
+  });
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  expect(res?.response).toBeDefined();
+  expect(res?.logs).toBeDefined();
+  expect(res?.logs.includes('hello world')).toBeTruthy();
+  expect(res?.success).toBeTruthy();
+};
+
+const executeJsSigning = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  const alice = await devEnv.createRandomPerson();
+  const litActionSessionSigs = await generator(devEnv, alice, [
+    {
+      resource: new LitPKPResource('*'),
+      ability: LitAbility.PKPSigning,
+    },
+    {
+      resource: new LitActionResource('*'),
+      ability: LitAbility.LitActionExecution,
+    },
+  ]);
+
+  const res = await devEnv.litNodeClient?.executeJs({
+    sessionSigs: litActionSessionSigs,
+    code: `(async () => {
+      const sigShare = await LitActions.signEcdsa({
+        toSign: dataToSign,
+        publicKey,
+        sigName: "sig",
+      });
+    })();`,
+    jsParams: {
+      dataToSign: alice.loveLetter,
+      publicKey: alice.authMethodOwnedPkp?.publicKey,
+    },
+  });
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  expect(res?.signatures?.sig.r).toBeDefined();
+  expect(res?.signatures?.sig.s).toBeDefined();
+  expect(res?.signatures?.sig.dataSigned).toBeDefined();
+  expect(res?.signatures?.sig.publicKey).toBeDefined();
+};
+
+const executeJsSigningParallel = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
```

Comment:
🔪 ideally this would be replaced with wrapping the test definition function call itself in a conditional check, see above comment.

That being said, do we _really_ never run any test that uses this function on manzano -- and should it be disabled on `datil-test` if it was disabled on `manzano`?